drizzle-orm/src/postgres-js/driver.ts

```diff
@@ -27,10 +27,20 @@ function construct<TSchema extends Record<string, unknown> = Record<string, neve
 } {
 	const transparentParser = (val: any) => val;
 
+	const dateSerializer = (val: unknown): unknown => {
+		if (Object.prototype.toString.call(val) === '[object Date]') {
+			return (val as Date).toISOString();
+		}
```

Comment:
Why not use `val instanceof Date`?

Reply:
This is safer because `instanceof` might return false if the Date object comes from another realm.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/instanceof#instanceof_and_multiple_realms